src/avram/criteria_extensions/includes_criteria.cr

```diff
@@ -0,0 +1,27 @@
+module Avram::IncludesCriteria(T, V)
+  macro included
+    # WHERE `value` = ANY(column)
+    def includes(value) : T
+      value = V.adapter.to_db!(value)
+      add_clause(Avram::Where::Includes.new(column, value))
+    end
+
+    # TODO: Figure out how we can ensure this method
+    # is only called on array columns at compile-time.
+    private def check_using_array!(_value : Array.class)
+    end
+
+    private def check_using_array!(_value)
```

Comment:
These aren't actually used right now, but I wanted to leave them in as a template to the compile-time pattern. I do plan on eventually figuring out how to use it.